attributes/default.rb

```diff
@@ -28,23 +28,38 @@
   ["django-bcrypt", "0.9.2"],
   ["django-sendmail-backend", "0.1.2"],
 ]
+# dependencies per: https://docs.getsentry.com/on-premise/server/installation/python/
 default["sentry"]["dependency"]["packages"] = [
+  "python-setuptools",
+  "python-pip",
```

Comment:
You probably don't `python-pip` or `python-setuptools`.
The [python](https://github.com/poise/python) cookbook should take care of that for you.